custom/src/test/java/io/lumigo/javaagent/common/SecretScrubberTest.java

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright 2023 Lumigo LTD
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package io.lumigo.javaagent.common;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import io.opentelemetry.sdk.autoconfigure.spi.ConfigProperties;
+import java.time.Duration;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+
+public class SecretScrubberTest {
+  @Test
+  public void testEmptyEnvironmentMasking() {
+    ConfigProperties config =
```

Comment:
Maybe mockito can make this less verbose?

Reply:
Now added